superset/assets/src/dashboard/deprecated/v1/components/Controls.jsx

```diff
@@ -144,6 +144,7 @@ class Controls extends React.PureComponent {
             }
           />
           {dashboard.dash_save_perm &&
+            dashboard.force_v2_edit &&
```

Comment:
should this be negated? `!dashboard.force_v2_edit`

Reply:
force_v2_edit  = CAN_FALLBACK_TO_DASH_V1_EDIT_MODE is false